chia/wallet/puzzles/dao_proposal.clvm

```diff
@@ -14,6 +14,7 @@
 
```

Comment:
I'm actually nor sure this puzzle is doing a lineage proof to some original state which is bad news right because someone an just create an already completed proposal out of thin air?